_topic_map.yml

```diff
@@ -217,6 +217,16 @@ Topics:
         Distros: openshift-enterprise,openshift-origin,openshift-dedicated
       - Name: Other API Objects
         File: other_api_objects
+  - Name: Service Catalog Components
+    Dir: service_catalog
+    Distros: openshift-enterprise,openshift-origin
+    Topics:
+      - Name: Service Catalog
+        File: index
+      - Name: Template Service Broker
+        File: template_service_broker
+      - Name: Ansible Service Broker
```

Comment:
I don't think we make such a distinction right now, but long term, there is likely to be a distinction between an ansible broker that you can run yourself with your own playbooks, and the instance of the ansible broker which pulls APBs from the RHCC